tests/dummy/config/environment.js

```diff
@@ -5,7 +5,7 @@ module.exports = function(environment) {
     modulePrefix: 'dummy',
     environment,
     rootURL: '/',
-    locationType: 'auto',
+    locationType: 'trailing-history',
```

Comment:
What does this change do?

Reply:
Update: enforces trailing slash `/` for all urls to make relative links work properly in markdown